soroban-settings/testnet_settings_phase_5.json

```diff
@@ -0,0 +1,47 @@
+{
+    "updated_entry": [
```

Comment:
You don't need to add a separate file for testnet. You can just update `testnet_settings_phase_2.json` so it can be used after a testnet reset.

Reply:
Good point. Since testnet gets reset we don't need to retain history.
How about having a single file for Testnet without any numeric prefix?

Reply:
We unfortunately need the phase 1 file to update `contract_data_entry_size_bytes` first because the full upgrade exceeds the initial limit.

Reply:
How about renaming the testnet files to something like,
```
testnet_settings_enable_upgrades.json
testnet_settings_phase_n.json
```

Reply:
Sure. We could probably just remove n. It can be something like

```
testnet_settings_enable_upgrades.json
testnet_settings.json or testnet_settings_upgrade.json
```

Reply:
Done.